internal/pkg/template/templates/workloads/partials/cf/sidecars.yml

```diff
@@ -44,8 +44,8 @@
 {{- if $sidecar.Essential}}
   Essential: {{$sidecar.Essential}}
 {{- end}}
+{{include "image-overrides" . | indent 2}}  
```

Comment:
:cool_crying:

Would it be possible to add another sidecar here without `port` to make sure we have a case covering it 🥺: 
https://github.com/aws/copilot-cli/blob/a633641af728dadba33aa93bbb53491d5898037d/internal/pkg/deploy/cloudformation/stack/testdata/workloads/svc-manifest.yml#L81-L89